### PR TITLE
adds help text for types

### PR DESCRIPTION
**Pull request by katiedavis**
Screen Shot 2019-10-09 at 10 42 39 AM

Lets the user know they can pass type

### CODE REVIEW

File: lib/shopify-cli/commands/generate.rb [hunk 1]
```diff
@@ -23,8 +23,9 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:page}}: Generate a new page in your app with the specified page name.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}}
-
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}} or
```
Comment (gfscott) [inline lib/shopify-cli/commands/generate.rb]: Just thinking ahead about the scalability of this model, where under "Usage" we list possible combinations of commands, separated by "or" at the end of the line. What happens if we add more options? Just keep adding lines? Or is there a more extensible content model?

To be clear, I think this is **totally good to go today**, but could we make an issue to investigate how to grow this model sustainably in the longer term?

File: lib/shopify-cli/commands/generate.rb [hunk 2]
```diff
@@ -23,8 +23,9 @@ def self.extended_help
...
+                     {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename> --type=TYPE}}
+              Types:
               {{cyan:empty-state}}: generate a new page with an empty state
               {{underline:https://polaris.shopify.com/components/structure/empty-state}}
```
Comment (gfscott) [inline lib/shopify-cli/commands/generate.rb]: Our usual style is to indent everything under a cyan command to aid scannability. Can you indent the URLs?

File: lib/shopify-cli/commands/generate.rb [hunk 3]
```diff
@@ -23,8 +23,9 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:page}}: Generate a new page in your app with the specified page name.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}}
-
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename>}} or
+                     {{command:#{ShopifyCli::TOOL_NAME} generate page <pagename> --type=TYPE}}
+              Types:
               {{cyan:empty-state}}: generate a new page with an empty state
```
Comment (gfscott) [inline lib/shopify-cli/commands/generate.rb]: The explanatory text after the colon should be in sentence case — capitalized with punctuation at the end!

File: lib/shopify-cli/commands/generate.rb (hunk 1 above)
Comment (davekilljoy) [reply lib/shopify-cli/commands/generate.rb]: Agree, I told Katie I'd look into a longer term/scalable design for this type of help content.

**Pull request merged**